src/collection/collection.service.ts

```diff
@@ -1,11 +1,13 @@
 import { ExecutionResult, MutationFunction } from '@apollo/react-common';
```

Comment:
In deze file zijn vooral wat comments verplaatst en CustomErrors rond errors gehangen. Aan de functionaliteit is niets veranderd.